pkg/restclient/volume.go

```diff
@@ -90,9 +90,12 @@ func (c *Client) VolumeStatus(volname string) (api.VolumeStatusResp, error) {
 }
 
 // VolumeStart starts a Gluster Volume
-func (c *Client) VolumeStart(volname string) error {
+func (c *Client) VolumeStart(volname string, force bool) error {
+	req := api.VolumeStartReq{
+		Force: force,
```

Comment:
`Force` is too broad. Now that this in request body, we can name it more specifically. Something like - `ForceStartBricks` (var name) and `force-start-bricks` (in json)

Reply:
@prashanthpai done